commit 83a8da644b553680e9d578843a6fa0fc9347af0e
Author: J. Bruce Fields <bfields@citi.umich.edu>
Date:   Thu Aug 28 11:25:29 2008 -0400

    Remove idmapd.conf
    
    The example idmapd.conf file is kept in libnfsidmap now, which is what's
    responsible for parsing it anyway.
    
    Signed-off-by: J. Bruce Fields <bfields@citi.umich.edu>
    Signed-off-by: Steve Dickson <steved@redhat.com>

diff --git a/utils/idmapd/idmapd.conf b/utils/idmapd/idmapd.conf
deleted file mode 100644
index acafd4b..0000000
--- a/utils/idmapd/idmapd.conf
+++ /dev/null
@@ -1,10 +0,0 @@
-[General]
-
-Verbosity = 0
-Pipefs-Directory = /var/lib/nfs/rpc_pipefs
-Domain = localdomain
-
-[Mapping]
-
-Nobody-User = nobody
-Nobody-Group = nobody
